src/program/program.spec.ts

```diff
@@ -370,4 +370,17 @@ describe("Collection", () => {
 	test("should populate previous and next based on module ID", function () {
 		expect(true).toBe(true);
 	});
+
+	test("should set first content entry of a lesson as primary content entry", async function () {
+		const coll = await resolver.collection({ id: testingCollectionID });
+		expect(coll).toBeDefined();
+		if (coll.length > 0) {
+			coll.map((c) => {
+				c.lessons.map((lesson) => {
+					if (lesson.content.length == 1)
+						expect(lesson.content[0].primary).toBe(true);
```

Comment:
We are expecting the lesson's first content to be primary if the array length is 1. 
- What is this test cases checking if the content length is more then 1?
- Is this test case covering creating content with a primary conflict?
- Is this test case covering updating content to be primary where there already is a primary in the Array?